spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java

```diff
@@ -37,7 +37,11 @@
  */
 public class ModuleUtils {
 
-	private static final AsciiBytes LIB = new AsciiBytes("lib/");
+	private static final String LIB = "lib/";
+
+	public static final String DOT_JAR = ".jar";
+
+	public static final String DOT_ZIP = ".zip";
```

Comment:
My bad, I was under the impression we only looked up .jar
So yeah, we shall accept .zip
The more, the merrier...